tests/aws/services/s3/test_s3_list_operations.py

```diff
@@ -166,34 +166,6 @@ def test_list_buckets_region_validation(self, aws_client, snapshot):
             aws_client.s3.list_buckets(BucketRegion="eu-east-1")
         snapshot.match("bad-region", e.value.response)
 
-    @markers.aws.only_localstack
-    @markers.requires_in_process
-    def test_list_buckets_region_validation_disabled(
-        self, snapshot, monkeypatch, aws_client_factory, s3_create_bucket_with_client
-    ):
```

Comment:
note: I believe we might want to keep this test, and have a special case in the location constraint validation to skip it if `config.ALLOW_NONSTANDARD_REGIONS` is set, we can have a quick call about this to give you more context if you'd like!